lnst/Recipes/ENRT/SRIOVNetnsOvSRecipe.py

```diff
@@ -0,0 +1,197 @@
+        """
+        host1, host2 = self.matched.host1, self.matched.host2
+        configuration = super().test_wide_configuration()
+        configuration.test_wide_devices = []
+
+        for i, host in enumerate([host1, host2]):
+            host.run("systemctl enable openvswitch")
+            host.run("systemctl start openvswitch")
+            host.run("ovs-vsctl set Open_vSwitch . other_config:hw-offload=true")
+
+            host.run(f"devlink dev eswitch set pci/{host.eth0.bus_info} mode switchdev; sleep 2")
+            host.run(f"echo 1 > /sys/class/net/{host.eth0.name}/device/sriov_numvfs; sleep 5")
+
+            vf_ifname = dict(ifname=f"{host.eth0.name}v0")
+            host.map_device("vf_eth0", vf_ifname)
+
+            host.newns = NetNamespace(f"lnst{i + 1}")
+            host.newns.vf_eth0 = host.vf_eth0
+
+            vf_representor_ifname = dict(ifname="eth0")
+            host.map_device("vf_representor_eth0", vf_representor_ifname)
+            host.vf_representor = host.vf_representor_eth0
+
+            host.br0 = OvsBridgeDevice()
+            host.br0.port_add(host.eth0)
+            host.br0.port_add(host.vf_representor)
+
+            for dev in [host.eth0, host.newns.vf_eth0, host.vf_representor, host.br0]:
+                dev.up()
+
+            host.newns.vf_eth0.ip_add(ipaddress("192.168.101." + str(i + 1) + "/24"))
+            host.newns.vf_eth0.ip_add(ipaddress("fc00::" + str(i + 1) + "/64"))
+
+            configuration.test_wide_devices.append(host.newns.vf_eth0)
+
+        self.wait_tentative_ips(configuration.test_wide_devices)
+
+        return configuration
+
+    def generate_test_wide_description(self, config):
+        """
+        Test wide description consists of mirror setup of a single namespace,
+        containing virtual function. Representor of this virutal function, as well as,
+        physical function is plugged into the OvS bridge, which has hardware offloading
+        capabilities enabled.
+        """
+
+        desc = super().generate_test_wide_description(config)
+        host1, host2 = self.matched.host1, self.matched.host2
+        for i, host in enumerate([host1, host2]):
+            desc += [
+                f"Configured host{i} interface {host.eth0.name} to be controlled by the switchdev driver \n"
+                "and created a single virtual function via PCI SYS interface.\n"
+                f"Created OvS bridge {host.br0} with enabled hardware offloading functionality.\n"
+                f"Configured new namespace to mimic network functionality of container, where \n"
+                f"virtual function {host.newns.vf_eth0.name} was moved and had following IP addresses \n"
+                f"assigned: {host.newns.vf_eth0.ips}.\n"
+                f"Physical function {host.eth0} interface, as well as, virtual function representor \n"
+                f"interface {host.vf_representor} were plugged into the OvS bridge {host.br0}."
```

Comment:
Not necessarily need to be resolved now, but for this descriptions try to
* use short sentences - avoid 'and'
* split this to individual configurations made, so that they're each on one line - see other ENRT recipes e.g. lnst/Recipes/ENRT/VirtualBridgeVlansOverBondRecipe.py

Reply:
fixed